.github/ISSUE_TEMPLATE/0-bugissue.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Bug Issue
+about: Use this template for reporting a bug
+---
+
+## System information
+- CPU information(`cat /proc/cpuinfo | grep "model name" | uniq`):
+- GPU information(`lspci -nn | grep -E 'VGA|isplay`):
+
+## Issue behavior
+### Describe the current behavior
+
+### Describe the expected behavior
+
+## Debug information
+- What's libva/libva-utils/gmmlib/media-driver version?
+- Could you confirm whether GPU hardware exist or not by `ls /dev/dri`?
+- Could you provide vainfo log if possible by `vainfo >vainfo.log 2>&1`?
```

Comment:
if vainfo failed, how about vainfo --display drm ,  if there are multiple display device, please try vainfo --display xx --device xx

Reply:
Let's suggest customer to specify the device if we see there are multiple device in environment.